src/main/java/org/terasology/structureTemplates/internal/systems/StructureSpawnServerSystem.java

```diff
@@ -252,8 +262,9 @@ public void setConstructionComplete(DelayedActionTriggeredEvent event, EntityRef
         if (!event.getActionId().equals(CONSTRUCTION_COMPLETE_ACTION_ID)) {
             return;
         }
-
-        structureEntity.send(new StructureBlocksSpawnedEvent(regionTransform));
+        // once the structure has been placed create the multiblock entity
+        EntityRef block = blockRegistry.getBlockEntityAt(location);
+        block.send(new SendRegionEvent(blockRegionList));
```

Comment:
Does this always create a multi-block entity for all structure templates now?

Reply:
I think this is restricted to StructureTemplates being spawned using a layered animation. So in case we want this to happen for other kind of animations or no animation we would have to send those regions separately.

Reply:
I think it is an inconsistency that this happens only for a specific animation, especially since animation and multi-block have essentially nothing to do with each other.

To be honest, I'd like to derive whether to create a multi-block structure or not from the structure template itself, i.e., only send this event if the template states that it shall create a multi-block structure. 
Such information could even be paired with the _type_ that should be used for the spawned structure.

Reply:
Looks like the other animations don't work perfectly and the game crashes and doesn't work as expected when trying to use the other animations (FallingBlocks and NoAnimation) ,  only the default layered animation works perfectly for me . Most of the Structures use the default Layered Animation so multiBlock should work for all of them. Not too sure where the problem lie's yet though.